<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uiMagician..</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AutosResourceFile.qrc">
    <normaloff>:/Images/Images/fatbee_v2.ico</normaloff>:/Images/Images/fatbee_v2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Room</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="roomLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Floor  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="floorLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>House</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="houseLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="testCaseTableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>444</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Room/Zone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testCaseNewPushButton">
              <property name="text">
               <string>New</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/NEW1.BMP</normaloff>:/Images/Images/NEW1.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testCaseSavePushButton">
              <property name="text">
               <string>Save</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/SAVE.BMP</normaloff>:/Images/Images/SAVE.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testCaseDeletePushButton">
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/DELETE.BMP</normaloff>:/Images/Images/DELETE.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="roomUndoPushButton">
              <property name="text">
               <string>Undo</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/UNDO.BMP</normaloff>:/Images/Images/UNDO.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="roomRedoPushButton">
              <property name="text">
               <string>Redo</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/REDO.BMP</normaloff>:/Images/Images/REDO.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Configuration File</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>4</width>
                <height>30</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="testReportNewPushButton">
              <property name="text">
               <string>New</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/NEW1.BMP</normaloff>:/Images/Images/NEW1.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testReportOpenPushButton">
              <property name="text">
               <string>Open</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/OPEN.BMP</normaloff>:/Images/Images/OPEN.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testReportSavePushButton">
              <property name="text">
               <string>Save</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/SAVE.BMP</normaloff>:/Images/Images/SAVE.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testReportClosePushButton">
              <property name="text">
               <string>Close</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/DELETE.BMP</normaloff>:/Images/Images/DELETE.BMP</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Device       </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testStepNewPushButton">
              <property name="text">
               <string>New</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/NEW1.BMP</normaloff>:/Images/Images/NEW1.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testStepDeletePushButton">
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/DELETE.BMP</normaloff>:/Images/Images/DELETE.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deviceUndoPushButton">
              <property name="text">
               <string>Undo</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/UNDO.BMP</normaloff>:/Images/Images/UNDO.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="devicRedoPushButton">
              <property name="text">
               <string>Redo</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/REDO.BMP</normaloff>:/Images/Images/REDO.BMP</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>168</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="testReportVerifyPushButton">
              <property name="text">
               <string>Verify</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/Gears.ico</normaloff>:/Images/Images/Gears.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="appGeneratePushButton">
              <property name="text">
               <string>Generate App</string>
              </property>
              <property name="icon">
               <iconset resource="AutosResourceFile.qrc">
                <normaloff>:/Images/Images/Gears.ico</normaloff>:/Images/Images/Gears.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>145</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContent_Help_F1"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Ui_Magician"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="leftTreeDockWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>Room / Zone</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeWidget" name="treeWidgetLeft">
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/NEW.BMP</normaloff>
     <normalon>:/Images/Images/NEW.BMP</normalon>:/Images/Images/NEW.BMP</iconset>
   </property>
   <property name="text">
    <string>New                Ctrl+N</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/OPEN.BMP</normaloff>:/Images/Images/OPEN.BMP</iconset>
   </property>
   <property name="text">
    <string>Open              Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/SAVE.BMP</normaloff>:/Images/Images/SAVE.BMP</iconset>
   </property>
   <property name="text">
    <string>Save                Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionContent_Help_F1">
   <property name="text">
    <string>Content Help     </string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/DELETE.BMP</normaloff>:/Images/Images/DELETE.BMP</iconset>
   </property>
   <property name="text">
    <string>Close Current Project          </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit_Ctrl_Q">
   <property name="text">
    <string>Exit                  Ctrl+Q</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/UNDO.BMP</normaloff>:/Images/Images/UNDO.BMP</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="AutosResourceFile.qrc">
     <normaloff>:/Images/Images/REDO.BMP</normaloff>:/Images/Images/REDO.BMP</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout_Ui_Magician">
   <property name="text">
    <string>About Ui Magician....</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutosResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
